source/sass/pages/ways-to-give-page.scss

```diff
@@ -0,0 +1,42 @@
+.app--ways-to-give {
+  .app__header {
+    background: $color--black;
+    color: $color--white;
+    height: 300px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    .button {
+      display: inline-block;
+      text-transform: uppercase;
+    }
+  }
+
+  .container {
+    padding: 2em 10px;
+    max-width: 960px;
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  h1 {
+    @include font-size(xxxl);
+    margin: 0 0 20px;
+  }
+
+  h2 {
+    @include font-size(xxl);
+  }
+
+  h3 {
+    @include font-size(xl);
+  }
+
+  ul {
```

Comment:
It might be nice to add a `column-count: 2` (or 3) on this to make the page shorter, but that's just a random thought.